src/bp/core/services/stats/index.ts

```diff
@@ -0,0 +1,125 @@
+import axios from 'axios'
+import LicensingService from 'common/licensing-service'
+import { machineUUID } from 'common/stats'
+import { TYPES } from 'core/types'
+import { inject, injectable } from 'inversify'
+import ms from 'ms'
+import path from 'path'
+
+import { GhostService } from '..'
+import { BotService } from '../bot-service'
+import { JobService } from '../job-service'
+
+const LOCK_RESOURCE = 'botpress:statsService'
+@injectable()
+export class StatsService {
+  constructor(
+    @inject(TYPES.JobService) private jobService: JobService,
+    @inject(TYPES.BotService) private botService: BotService,
+    @inject(TYPES.GhostService) private ghostService: GhostService,
+    @inject(TYPES.LicensingService) private licenseService: LicensingService
+  ) {}
+
+  public start() {
+    // tslint:disable-next-line: no-floating-promises
+    this.run()
+    setInterval(this.run.bind(this), ms('6 hours'))
+  }
+
+  private async run() {
+    const lock = await this.jobService.acquireLock(LOCK_RESOURCE, ms('1 minute'))
+    if (lock) {
+      await this.sendStats()
+    }
+  }
+
+  private async sendStats() {
+    const stats = await this.getStats()
+    try {
+      await axios.post('https://telemetry.botpress.io/ingest', stats)
+    } catch (err) {
+      // silently fail
+    }
+  }
+
+  private async getStats() {
+    return {
+      schema: '1.0.0',
+      timestamp: new Date().toISOString(),
+      bots: {
+        count: await this.getBotsCount()
+      },
+      flows: {
+        count: await this.getFlowCount()
+      },
+      intents: {
+        count: await this.getIntentsCount()
+      },
+      contentElements: {
+        count: await this.getContentElementsCount()
+      },
+      server: {
+        externalUrl: process.EXTERNAL_URL || `http://${process.HOST}:${process.PORT}`,
+        botpressVersion: process.BOTPRESS_VERSION,
+        fingerprint: await this.getServerFingerprint(),
+        clusterEnabled: !(process.CLUSTER_ENABLED === null),
```

Comment:
I'd add machine OS, total RAM, languages used, nb of QnAs vs intents, NB of entities, NB of collaborators, Pro vs Community, uptime since last restart, NB of restart, total uptime, NB of hooks (global vs per-bot), NB of actions (global vs per-bot), pipeline enabled or not, NB of workspaces, # of chat users in last 30 days per channel

Reply:
Also: if showPoweredBy is enabled in config, the number of super admins, the number of auth strategies configured,